patient-portal-ui/src/datajs/DeafutlAllData.tsx

```diff
@@ -1,14 +1,14 @@
 import React, { useState, useEffect } from "react";
 import { dateIsoToString } from '../utils/ManageDate';
-const url_0 = 'http://api-develop.ohpp.local:18080/api/';
-const url_1 = 'http://api-develop.ohpp.local:18080/api/';
+// const url_0 = 'http://develop-api.ohpp.local:18080/api/';
+const url_0 = 'http://localhost:18080/api/'; // --- uncomment to develop locally
 // export const ListDoctorPatientsTest = async () => {
 export const DeafutlAllData = {
 
 
   // --- ADMIN_API
   getAllUsers: async function () {
-    let response = await fetch('http://api-develop.ohpp.local:18080/api/admin/users');
+    let response = await fetch('http://develop-api.ohpp.local:18080/api/admin/users');
```

Comment:
the correct form is `api-develop.ohpp.local`, but I think you can reuse url_0 here